job-dsls/jobs/kogito_daily.groovy

```diff
@@ -0,0 +1,134 @@
+                checkout([$class: 'GitSCM', branches: [[name: '$baseBranch']], browser: [$class: 'GithubWeb', repoUrl: 'https://github.com/$organization/kogito-runtimes'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kogito-runtimes']], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/$organization/kogito-runtimes.git']]])
+                checkout([$class: 'GitSCM', branches: [[name: '$baseBranch']], browser: [$class: 'GithubWeb', repoUrl: 'https://github.com/$organization/kogito-cloud'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kogito-cloud']], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/$organization/kogito-cloud.git']]])
+                checkout([$class: 'GitSCM', branches: [[name: '$baseBranch']], browser: [$class: 'GithubWeb', repoUrl: 'https://github.com/$organization/kogito-examples'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kogito-examples']], submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/$organization/kogito-examples.git']]])                
+            }
+        }
+        stage ('Remove M2') {
+            steps {
+                sh 'if [ -d $m2Dir ]; then \\n' +
+                        'rm -rf $m2Dir/org/kogito \\n' +
+                    'fi'
+            }
+        }         
+        stage('Update versions') {
+            steps {
+                sh "echo 'kogitoVersion: $kogitoVersion'"
+                sh "sh kogito-bom/scripts/release/02-update-version-all.sh $kogitoVersion"
+            }
+        }
+        stage('Install repositories'){
+            steps {
+                configFileProvider([configFile(fileId: '771ff52a-a8b4-40e6-9b22-d54c7314aa1e', targetLocation: 'jenkins-settings.xml', variable: 'SETTINGS_XML_FILE')]) {
+                    sh "cd kogito-bom && mvn -B -e -U clean install -Dfull -s $SETTINGS_XML_FILE -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true"
+                    sh "cd kogito-runtimes && mvn -B -e -U clean install -Dfull -s $SETTINGS_XML_FILE -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true"
+                    sh "cd kogito-cloud && mvn -B -e -U clean install -Dfull -s $SETTINGS_XML_FILE -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true"
+                    sh "cd kogito-examples && mvn -B -e -U clean install -Dfull -s $SETTINGS_XML_FILE -Dmaven.test.redirectTestOutputToFile=true -Dmaven.test.failure.ignore=true"
```

Comment:
I have some doubts now if we shouldn't deploy instead of install into snapshots.jboss.org

WDYT @mswiderski  @krisv ?

Reply:
@mareknovotny there is already a deploy job https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/kogito-deploy/

Reply:
ok, so it is split in more jobs, thanks for quick answer.